src/migrations/2017-08-10-organization_uuid.js

```diff
@@ -0,0 +1,10 @@
+import { r } from '../server/models'
+
+async function migrate() {
```

Comment:
It doesn't really matter if the ID is exposed, it just can't be used to join the org because it's easily guessable. And joining with UUID instead requires getting the UUID from the organization admin to the texter and then back to the app. URL is the easiest way to transmit the UUID.